lib/money/currency.rb

```diff
@@ -267,7 +265,7 @@ def stringify_keys
     # @example
     #   Money::Currency.new(:usd) #=> #<Money::Currency id: usd ...>
     def initialize(id)
-      @id = id.to_sym
+      @id = id&.to_sym
```

Comment:
As far as I can see this should never get initialized with a `nil`. What's the use case for this?

Reply:
I just realised that `Money::Currency.new` is still called when running `Money.default_currency`. Made the necessary changes in `.default_currency` to remove the safe navigation operator in Currency's `initialize` method.